README.md

```diff
@@ -1,22 +1,97 @@
+
+### Execute Glue job
+AWS Glue is used to generate a combined table from the import and export Athena tables. To do this, there is a job (stored and deployed from aws-ingestion repo) which gets the data from the Athena tables and performs pyspark jobs on the data to generate a full table of all records. Matching ids from both tables go on the same row and calculations like which manifest set has the earliest timestamp, the types of records and other things are worked out in the glue job and added to the table. The results are saved are perquet files (a format similar to json but more performant for big data queries). The glue job produces another Athena table for the combined data and this acts like the others. The difference is that this table sits over the parquet files in S3. If they were json format all the sql queries we run would time out.
+
+### Generate the queries
+In this repo, there are lots of SQL queries which we use to run against the combined table and produce the reports. See the top of this document for the explanations of each query output. Technically, these are Presto SQL queries as Athena runs Presto SQL under the hood. This is slightly different to Oracle syntax, MS SQL syntax or other SQL flavours but has documentation on the internet to help.
+
+The queries are split in to different types and can be found in folders with the same name as the type.
+
+### Production queries
+These are the queries that make up the main report and are always run.
+
+### Import queries
+These are queries which tell us information about the import data only. We are not as concerned about the results of these as they are not something to be concerned about with missing data and more for information only. These don't run with the main production manifest, but can be run on their own.
+
+### Specific queries
+These are queries purely for information only and they help with debugging. They can only be run by themselves.
+
+## Alerts
+For further detail, see:
+[Kafka reconciliation - missing records](https://git.ucd.gpn.gov.uk/dip/aws-common-infrastructure/wiki/DataWorks-AWS-Corporate-Storage#kafka-reconciliation---missing-records)
+[Manifest glue job failed](https://git.ucd.gpn.gov.uk/dip/aws-common-infrastructure/wiki/DataWorks-AWS-Corporate-Storage#manifest-glue-job-failed)
+
+## Legacy
+The [behave framework](https://github.com/dwp/dataworks-behavioural-framework) currently operates the reconciliation on the `manifest-comparison` ConcourseCI pipeline. The new kafka reconciliation can be triggered by starting the `corporate-storage-coalescer` pipeline.
```

Comment:
```suggestion
The [behave framework](https://github.com/dwp/dataworks-behavioural-framework) used to be used for the kafka reconciliation process (then known as the `manifest comparison` process) and all that code still exists and works and uses the `manifest-comparison` ConcourseCI pipeline.

In the future, it would be good to move relevant parts to the new process (mainly the e2e jobs) and remove the remaining legacy tasks.
```